syntax = "proto3";
package hiPb;

enum TxType{
  EVM = 0;
  DATA = 1;
}

message Tx {
  TxType type = 1;
  bytes from_addr = 2;
  bytes to_addr = 3;
  bytes data = 4;
  uint64 timestamp = 5;
  uint64 nonce = 6;
  bytes hash = 7;
}
